openbb_sdk/providers/benzinga/openbb_benzinga/models/stock_news.py

```diff
@@ -18,8 +18,11 @@ class BenzingaStockNewsQueryParams(StockNewsQueryParams):
     Source: https://docs.benzinga.io/benzinga/newsfeed-v2.html
     """
 
-    class Config:
-        fields = {"symbols": "tickers", "limit": "pageSize"}
+    pageSize: int = Field(
```

Comment:
Here think we can just use the alias dict and map "limit" to "pageSize", then in `extract_data` we `model_dump(by_alias=True)` ?

Reply:
That would be the preferred method for the standardization to work.